submissions/dashakim/oop-task/js/app.js

```diff
@@ -0,0 +1,100 @@
+//game score
+var score = 0
+//Enemy class with update(), render() methods
+var Enemy = function (x, y, speed) {
+    this.x = x
+    this.y = y
+    this.speed = speed
+    this.sprite = 'images/car.png';
+};
+var END_CANVAS = 510
+var BEGIN_CANVAS = -50
+Enemy.prototype.update = function (dt) {
+
+    this.x += this.speed * dt
+    if (this.x > END_CANVAS) {
+        this.x = BEGIN_CANVAS
+        //random speed for enemies
+        this.speed = 100 + Math.floor(Math.random() * 222)
+    }
+    // return to start position player when collide with enemy
+    if (player.x > (this.x - 60) && player.x < (this.x + 60) && player.y > (this.y - 60) && player.y < (this.y + 60)) {
```

Comment:
Using named constants like `END_CANVAS`, `BEGIN_CANVAS` is great for it gives numbers some semantic meaning and code reviewers like myself and possible contributors therefore can clearly understand what stands behind the numbers.

Unnamed constants like `60` here look mysterious. Please fix here and elsewhere across the code.

Reply:
You totally right!